Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ruby:2.6.6
```

Comment:
Why not 2.7?

Reply:
Because we are using `2.6.6` in the Gemfile at the moment and the `Dockerfile` should use the same 👍